diff --git a/qtwitter.pro b/qtwitter.pro
index bb5c548..4d38e55 100644
--- a/qtwitter.pro
+++ b/qtwitter.pro
@@ -41,8 +41,8 @@ RESOURCES += res/resources.qrc
 QT += network \
     xml
 
-# QTPLUGIN += qjpeg \
-# qgif
+QTPLUGIN += qjpeg \
+    qgif
 TRANSLATIONS += loc/qtwitter_pl.ts \
     loc/qtwitter_ca.ts \
     loc/qtwitter_es.ts
diff --git a/src/main.cpp b/src/main.cpp
index 5e5a44b..a554378 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -23,8 +23,8 @@
 #include "settings.h"
 #include "core.h"
 
-//Q_IMPORT_PLUGIN(qjpeg)
-//Q_IMPORT_PLUGIN(qgif)
+Q_IMPORT_PLUGIN(qjpeg)
+Q_IMPORT_PLUGIN(qgif)
 
 
 int main( int argc, char **argv )
