diff --git a/qtwitter.pro b/qtwitter.pro
index b3c7788..3fab3e0 100644
--- a/qtwitter.pro
+++ b/qtwitter.pro
@@ -40,8 +40,8 @@ RESOURCES += res/resources.qrc
 QT += network \
     xml
 
-# QTPLUGIN += qjpeg \
-# qgif
+QTPLUGIN += qjpeg \
+    qgif
 TRANSLATIONS += loc/qtwitter_pl.ts \
     loc/qtwitter_ca.ts \
     loc/qtwitter_es.ts
diff --git a/src/main.cpp b/src/main.cpp
index 1fbbb0e..8aa068e 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -23,8 +23,8 @@
 #include "settings.h"
 #include "core.h"
 
-//Q_IMPORT_PLUGIN(qjpeg)
-//Q_IMPORT_PLUGIN(qgif)
+Q_IMPORT_PLUGIN(qjpeg)
+Q_IMPORT_PLUGIN(qgif)
 
 
 int main( int argc, char **argv )
