diff --git a/qtwitter.pro b/qtwitter.pro
index c696fe0..ebab227 100644
--- a/qtwitter.pro
+++ b/qtwitter.pro
@@ -42,8 +42,8 @@ FORMS += ui/mainwindow.ui \
 RESOURCES += res/resources.qrc
 QT += network \
     xml
-#QTPLUGIN += qjpeg \
-#    qgif
+QTPLUGIN += qjpeg \
+    qgif
 TRANSLATIONS += loc/qtwitter_pl.ts \
     loc/qtwitter_ca.ts \
     loc/qtwitter_es.ts
diff --git a/src/main.cpp b/src/main.cpp
index f4d35b5..2d9b3b3 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -25,8 +25,8 @@
 #include <QSystemTrayIcon>
 #include <QSignalMapper>
 
-//Q_IMPORT_PLUGIN(qjpeg)
-//Q_IMPORT_PLUGIN(qgif)
+Q_IMPORT_PLUGIN(qjpeg)
+Q_IMPORT_PLUGIN(qgif)
 
 
 int main( int argc, char **argv )
