diff --git a/qtwitter.pro b/qtwitter.pro
index 00e2d7e..d7aa55a 100644
--- a/qtwitter.pro
+++ b/qtwitter.pro
@@ -46,8 +46,8 @@ FORMS += ui/mainwindow.ui \
 RESOURCES += res/resources.qrc
 QT += network \
     xml
-#QTPLUGIN += qjpeg \
-#    qgif
+QTPLUGIN += qjpeg \
+    qgif
 TRANSLATIONS += loc/qtwitter_pl.ts \
     loc/qtwitter_ca.ts \
     loc/qtwitter_es.ts
diff --git a/src/main.cpp b/src/main.cpp
index 3d130f3..28a7047 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -25,8 +25,8 @@
 #include <QSystemTrayIcon>
 #include <QSignalMapper>
 
-//Q_IMPORT_PLUGIN(qjpeg)
-//Q_IMPORT_PLUGIN(qgif)
+Q_IMPORT_PLUGIN(qjpeg)
+Q_IMPORT_PLUGIN(qgif)
 
 
 int main( int argc, char **argv )
