diff --git a/qtwitter.pro b/qtwitter.pro
index e9d0102..f454fa4 100644
--- a/qtwitter.pro
+++ b/qtwitter.pro
@@ -56,8 +56,8 @@ RESOURCES += res/resources.qrc
 QT += network \
     xml
 
-# QTPLUGIN += qjpeg \
-# qgif
+QTPLUGIN += qjpeg \
+    qgif
 TRANSLATIONS += loc/qtwitter_pl.ts \
     loc/qtwitter_ca.ts \
     loc/qtwitter_es.ts
diff --git a/src/main.cpp b/src/main.cpp
index ef2e383..06ece5a 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -26,8 +26,8 @@
 #include <QSystemTrayIcon>
 #include <QSignalMapper>
 
-//Q_IMPORT_PLUGIN(qjpeg)
-//Q_IMPORT_PLUGIN(qgif)
+Q_IMPORT_PLUGIN(qjpeg)
+Q_IMPORT_PLUGIN(qgif)
 
 
 int main( int argc, char **argv )
